### PR TITLE
Misc cleanup

### PR DESCRIPTION
**Pull request by kevmoo**
[Review on Reviewable](https://reviewable.io/reviews/dart-lang/dartdoc/995)

### CODE REVIEW

**Comment by kevmoo** [comment]
@keertip PTAL

File: lib/src/html_generator.dart [hunk 1]
```diff
@@ -221,61 +222,61 @@ class _HtmlGeneratorInstance implements HtmlOptions {
 
     generatePackage();
 
-    package.libraries.forEach((Library lib) {
+    for (var lib in package.libraries) {
```
Comment (keertip) [inline lib/src/html_generator.dart]: What is the reason for this change?

**Comment by keertip** [comment]
@sethladd, could you take a look at the template changes?

File: lib/src/html_generator.dart (hunk 1 above)
Comment (kevmoo) [inline lib/src/html_generator.dart]: By using for loops, we avoid creating extra closures, which add to call stacks.

There's also nothing to step into when debugging – just a good practice to encourage and model for our users.

**Comment by kevmoo** [comment]
@keertip @sethladd The template changes were just whitespace – and then adding the class attributes.

Would love a 2nd/3rd pair of eyes on em, though

File: lib/templates/_callable.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
-<dt id="{{htmlId}}" class="callable">
-    <span class="name {{#isDeprecated}}deprecated{{/isDeprecated}}">{{{linkedName}}}</span><span class="signature">(<wbr>{{{ linkedParamsNoMetadata }}})
-        <span class="returntype parameter">&#8594; {{{ linkedReturnType }}}</span>
-    </span>
+<dt id="{{htmlId}}" class="callable{{ #isInherited }} inherited{{ /isInherited}}">
```
Comment (sethladd) [inline lib/templates/_callable.html]: was this intentional to check for inherited here?

File: lib/templates/_property.html [hunk 3]
```diff
@@ -1,8 +1,8 @@
-<dt id="{{htmlId}}" class="property">
-    <span class="name">{{{linkedName}}}</span>
-    <span class="signature">&#8594; {{{ linkedReturnType }}}</span>
+<dt id="{{htmlId}}" class="property{{ #isInherited }} inherited{{ /isInherited}}">
```
Comment (sethladd) [inline lib/templates/_property.html]: again, checking this is intentional

**Comment by sethladd** [comment]
lgtm thanks!

File: lib/src/html_generator.dart (hunk 1 above)
Comment (keertip) [inline lib/src/html_generator.dart]: got it.

**Comment by keertip** [comment]
lgtm. Thanks for working on #641

File: lib/templates/_property.html (hunk 3 above)
Comment (kevmoo) [inline lib/templates/_property.html]: ditto

File: lib/templates/_callable.html (hunk 2 above)
Comment (kevmoo) [inline lib/templates/_callable.html]: Yup. May want to dim/hide both the `dd` and `dt`.
